_posts/2020-01-08-cockpit-210.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Cockpit 210
+author: kkoukiou
+date: 2020-01-08
+tags: cockpit linux
+slug: cockpit-210
+category: release
+summary: Cockpit with Overview, Dashboard and Machines improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 210.
+
+### Overview: Add CPU utilization bar to usage card
```

Comment:
```suggestion
### Overview: Add CPU utilization to usage card
```
Usage is more than just the bars. Sure, the bar is the main graphical component, but there are also statistics. (As mentioned on the call, we can have network utilization with just the current average up/down throughput without a bar, for example.)